cllc-public-app/Models.Extensions/Application.cs

```diff
@@ -163,7 +163,12 @@ public static void CopyValues(this MicrosoftDynamicsCRMadoxioApplication to, Vie
             to.AdoxioHascooleraccess = from.HasCoolerAccess;
 
             to.AdoxioLocatedaboveother = from.LocatedAboveOther;
-
+            
+            if(from.IsOnINLand == true){
+                to.AdoxioIsoninland = 845280000;
```

Comment:
This is just a nitpick, but would love to see a constant here instead of the hardcoded numeric value. 
Notice how we have to repeat the number below for the other side of the model <-> view-model conversion. 
If that number ever gets changed then there could be bugs if we forget to update both places. A constant seems cleaner to me. What do you think?

Reply:
I agree with Alejandro - this should be a constant or enum.